mteb/tasks/Clustering/fr/MLSUMClusteringS2S.py

```diff
@@ -13,7 +13,7 @@ class MLSUMClusteringS2S(AbsTaskClustering):
         name="MLSUMClusteringS2S",
         description="Clustering of newspaper article contents and titles from MLSUM dataset. Clustering of 10 sets on the newpaper article topics.",
         reference="https://huggingface.co/datasets/mlsum",
-        hf_hub_name="mteb/mlsum",
+        hf_hub_name="mteb/mlsum",  # can't find the original dataset on HF
```

Comment:
@imenelydiaker

Reply:
```suggestion
        hf_hub_name="mlsum", 
```
Link here: https://huggingface.co/datasets/mlsum